presto-verifier/src/test/java/com/facebook/presto/verifier/framework/TestDataVerification.java

```diff
@@ -357,6 +364,18 @@ public void testExecutionTimeSessionProperty()
         assertEvent(event.get(), SUCCEEDED, Optional.empty(), Optional.empty(), Optional.empty());
     }
 
+    @Test
+    public void testRunningInQueryBankMode()
```

Comment:
can you add similar tests for TestExplainVerification , TestCreateTableVerification etc

Reply:
Added.